psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -107,15 +107,9 @@ def run_sapt_dft(name, **kwargs):
     core.IO.set_default_namespace('dimer')
     data = {}
 
-    if (core.get_global_option('SCF_TYPE') == 'DF'):
```

Comment:
It won't let me do this as a suggestion, but I suggest all these deleted lines be restored and `== 'DF'` be replaced by `== 'DISK_DF'`